README.md

```diff
@@ -59,23 +59,21 @@ Remote is meant to allow people from anywhere (or at least anywhere in a given t
 | Dow Jones | On a case by case basis | ? | [Link](https://dowjones.jobs/) | ? |
 | Doyensec | Application Security Engineer | Full remote | [Link](https://www.careers-page.com/doyensec-llc) | ? |
 | Dragos | Some | US remote | [Link](https://careers.dragos.com/jobs) | ? |
-| Drift | Some | US remote | [Link](https://www.drift.com/about/careers/#openings) | ? |
 | Dropbox | All | Full Remote | [Link](https://jobs.dropbox.com/all-jobs) | ? |
 | Elastic | Some roles | Full remote | [Link](https://www.elastic.co/about/careers/#engineering) | ? |
 | Enable Security | Security | Full remote | [Link](https://de.linkedin.com/company/enablesecurity) | ? |
 | ExtraHop | All | Full Remote | [Link](https://www.extrahop.com/company/careers/) | ? |
-| Facebook | Some roles | Full remote | [Link](https://www.facebook.com/careers/) | ? |
 | Fastly | All | Full remote | [Link](https://www.fastly.com/about/careers) | ? |
 | Fieldguide | All | Full remote US | [Link](https://www.fieldguide.io/careers) | ? |
 | Figma | All | US/Canada remote | [Link](https://www.figma.com/careers) | No |
 | Fingerprint for success | All | Full remote | [Link](https://www.fingerprintforsuccess.com/careers) | ? |
 | Firebolt | Some areas | Full remote / Remote EU | [Link](https://www.firebolt.io/career) | ? |
-| Fireeye | Most jobs | Full remote / Remote US | [Link](https://www.fireeye.com/company/jobs.html) | ? |
+| FireEye (Trellix) | Most jobs | Full remote / Remote US | [Link](https://www.fireeye.com/company/jobs.html) | ? |
```

Comment:
Can be changed to `| Trellix | Some jobs | Partially Remote | [Link](https://careers.trellix.com/) | ? |` in the right alphabetically ordered position.